neo/Prompt/Commands/Show.py

```diff
@@ -167,3 +173,189 @@ def execute(self, arguments=None):
+                        return
+                except Exception:
+                    print("Could not find notifications from args: %s" % arguments)
+                    return
+
+            if len(events):
+                [print(json.dumps(e.ToJson(), indent=4)) for e in events]
+                return events
+            else:
+                print("No events found for %s" % item)
+                return
+        else:
+            print("Please specify a block index, address, or contract hash")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('block_index/address/contract_hash', 'the block, address, or contract to show notifications for')
+        return CommandDesc('notifications', 'show specified contract execution notifications', [p1])
+
+
+class CommandShowAccount(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+        if item is not None:
+            account = Blockchain.Default().GetAccountState(item, print_all_accounts=True)
+
+            if account is not None:
+                print(json.dumps(account.ToJson(), indent=4))
+                return account.ToJson()
+            else:
+                print("Account %s not found" % item)
+                return
+        else:
+            print("Please specify an account address")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('address', 'the address to show')
+        return CommandDesc('account', 'show the assets (NEO/GAS) held by a specified address', [p1])
+
+
+class CommandShowAsset(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+        if item is not None:
+            if item.lower() == "all":
+                assets = Blockchain.Default().ShowAllAssets()
+                print("Assets: %s" % assets)
+                return assets
+
+            if item.lower() == 'neo':
+                assetId = Blockchain.Default().SystemShare().Hash
+            elif item.lower() == 'gas':
+                assetId = Blockchain.Default().SystemCoin().Hash
+            else:
+                try:
+                    assetId = UInt256.ParseString(item)
+                except Exception:
+                    print("Could not find asset from args: %s" % arguments)
+                    return
+
+            asset = Blockchain.Default().GetAssetState(assetId.ToBytes())
+
+            if asset is not None:
+                print(json.dumps(asset.ToJson(), indent=4))
+                return asset.ToJson()
+            else:
+                print("Asset %s not found" % item)
+                return
+        else:
+            print("Please specify an asset hash or name")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('name/assetId/all', 'the name or assetId of the asset, or "all" shows all assets\n\n')
```

Comment:
the extra help is not shown because the `)` terminates the `description` string after the first line